### ISSUE TITLE
[feature request] Option to use REGEX to rename files.

### ISSUE THREAD

**Issue by jcruiser89** [dazedcat19/FMD#1451]
The file names change often and some mangas have extremely long names that do not display properly in manga reader apps.  I would like a way to set the file name to just the chapter number or whatever i like by using regex to parse the downloaded file name and having the series name as a parameter that can be used.

ex.  series name Berserk
chapter being downloaded is 55
by default it may do something like 57 - Berserk V7 Ch.55
I would like to be able to change that to just Ch.55 or Berserk Ch.55.

Another example is a manga called Kouryakuhon o Kushi Suru Saikyou no Mahoutsukai ~ to wa Iwa Senai Oreryuu Mao Tobatsu Saizen Ruuto~

Its way to long to display in any reader so I would just use the chapter number.

Its fine if this is a global feature, would be nice if it could be changed per favorite.

**Comment by Slasar41** [comment]
This is the setting to do what you want.
![image](https://github.com/user-attachments/assets/f81eae04-a253-43bb-bd00-bf2cf0e5072e)

**Comment by jcruiser89** [comment]
No, Im asking for a complete name replacement.  Something like Sonarr.

![image](https://github.com/user-attachments/assets/ed199931-7bca-4bfb-8560-2cdbc737a728)
